compiler/rustc_parse/src/parser/diagnostics.rs

```diff
@@ -2957,13 +2954,17 @@ impl<'a> Parser<'a> {
         err
     }
 
-    pub fn is_diff_marker(&mut self, long_kind: &TokenKind, short_kind: &TokenKind) -> bool {
+    /// This checks if this is a git diff marker, one of
+    /// >>>>>
+    /// =====
+    /// <<<<<
```

Comment:
```suggestion
    ///
    /// * `>>>>>`
    /// * `=====`
    /// * `<<<<<`
```

This gets interpreted as markdown resulting in the following docs:

![Screenshot 2024-05-14 at 17-35-24 f in unq - Rust](https://github.com/rust-lang/rust/assets/14913065/4512bb62-9906-4a9b-a015-f27bb2cfc4d0)

Needs quoting and should be turned into a bullet list.

Reply:
Minor: Note however that the doc comment you added does not accurately reflect what this method is doing. The token kinds are not hard coded to  `>`, `=` or `<` but they're soft coded as function parameters.

Imo, a slight rephrasing would improve this.